lib/reek/smells/attribute.rb

```diff
@@ -30,7 +30,7 @@ def self.contexts # :nodoc:
       end
 
       def self.default_config
-        super.merge(Core::SmellConfiguration::ENABLED_KEY => false)
+        super.merge(Core::SmellConfiguration::ENABLED_KEY => true)
```

Comment:
I'm thinking we sould enhance the reek_of matcher to automatically enable the smell to be deteced.